Link error while building on windows

I was trying to build neuron under msys2 with following Cmake command:

```
/mingw64/bin/cmake .. -G 'Unix Makefiles' \
   -DNRN_ENABLE_MPI=OFF \
   -DNRN_ENABLE_INTERVIEWS=OFF \
   -DNRN_ENABLE_PYTHON=OFF \
   -DNRN_ENABLE_RX3D=OFF \
   -DPYTHON_EXECUTABLE=/c/python354/python.exe \
   -DNRN_ENABLE_PYTHON_DYNAMIC=OFF \
   -DNRN_PYTHON_DYNAMIC='/c/python354/python.exe;/c/python368/python.exe' \
   -DCMAKE_INSTALL_PREFIX='/c/nrn-install' \
   -DCMAKE_C_COMPILER=/c/msys64x/mingw64/bin/cc.exe \
   -DCMAKE_CXX_COMPILER=/c/msys64x/mingw64/bin/g++.exe > cmake.log
make -j > make.log
```

This gives following error

```
kumbhar@DESKTOP-14JAHO1 MINGW64 ~/nrn/build_ext
$ make
[  5%] Built target nocmodl
[  5%] Update hh.mod for CoreNEURON compatibility
[  5%] Built target hh_update
[  5%] Built target nrnversion_h
[  5%] Linking CXX shared library ../../bin/libnrniv.dll
C:/msys64x/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: CMakeFiles/nrniv_lib.dir/objects.a(plot.c.obj):C:/msys64x/home/kumbhar/nrn/src/oc/plot.c:79: multiple definition of `hoc_plttext'; CMakeFiles/nrniv_lib.dir/objects.a(cygwinprt.c.obj):C:/msys64x/home/kumbhar/nrn/src/oc/mswinprt.c:351: first defined here
collect2.exe: error: ld returned 1 exit status
make[2]: *** [src/nrniv/CMakeFiles/nrniv_lib.dir/build.make:6649: bin/libnrniv.dll] Error 1
make[1]: *** [CMakeFiles/Makefile2:225: src/nrniv/CMakeFiles/nrniv_lib.dir/all] Error 2
make: *** [Makefile:150: all] Error 2
```

I still get `multiple definition of `hoc_plttext'` after enabling Interviews and Python. I see that there are two definitions in `src/oc/plot.c:79` and `src/oc/mswinprt.c:351`.

Also, different issue, with `make setup_exe`:

```
+ cd C:/nrn-install/demo/release
+ C:/nrn-install/lib/mknrndl2.sh
mod_func.c:1:10: fatal error: stdio.h: No such file or directory
    1 | #include <stdio.h>
      |          ^~~~~~~~~
compilation terminated.
make[4]: *** [C:/nrn-install/lib/mknrndll.mak:67: mod_func.o] Error 1
make[3]: *** [src/mswin/CMakeFiles/setup_exe.dir/build.make:80: src/mswin/CMakeFiles/setup_exe] Error 2
make[2]: *** [CMakeFiles/Makefile2:924: src/mswin/CMakeFiles/setup_exe.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:931: src/mswin/CMakeFiles/setup_exe.dir/rule] Error 2
make: *** [Makefile:483: setup_exe] Error 2
```

--- a/src/oc/mswinprt.c
+++ b/src/oc/mswinprt.c
@@ -348,7 +348,7 @@ void plprint(const char* s) {
 	printf("%s", s);
 }
 #endif
-//int hoc_plttext; //defined in plot.c
+
 #if !defined(__MWERKS__)
 int getpid() {
 #if 0

--- a/src/oc/mswinprt.c
+++ b/src/oc/mswinprt.c
@@ -348,7 +348,7 @@ void plprint(const char* s) {
 	printf("%s", s);
 }
 #endif
-int hoc_plttext;
+//int hoc_plttext; //defined in plot.c
 #if !defined(__MWERKS__)
 int getpid() {
 #if 0

